modules/manage/partials/iceberg/about-iceberg-topics.adoc

```diff
@@ -0,0 +1,476 @@
++
+[,bash,]
+----
+# Create new topic with five topic partitions, replication factor 1, and custom table partitioning for Iceberg
+rpk topic create <new-topic-name> -p5 -r1 -c "redpanda.iceberg.partition.spec=(<partition-key1>, <partition-key2>, ...)"
+----
++
+Valid `<partition-key>` values include a source column name or a transformation of a column. The columns referenced can be Redpanda-defined (such as `redpanda.timestamp`) or user-defined based on a schema that you register for the topic. The Iceberg table stores records that share the same partition key values in the same files based on this specification. 
++
+For example: 
++
+--
+* To partition the table by a single key, such as a column `col1`, use: `redpanda.iceberg.partition.spec=(col1)`. 
+* To partition by multiple columns, use a comma-separated list: `redpanda.iceberg.partition.spec=(col1, col2)`. 
+* To partition by the year of a timestamp column `ts1`, and a string column `col1`, use: `redpanda.iceberg.partition.spec=(year(ts1), col1)`.
+
+For details on the partitioning specification, including allowed transforms, see the https://iceberg.apache.org/spec/#partitioning[Apache Iceberg documentation^].
+--
+
+. Configure `redpanda.iceberg.mode` for the topic. You can choose one of the following modes:
++
+--
+* `key_value`: Creates an Iceberg table using a simple schema, consisting of two columns, one for the record metadata including the key, and another binary column for the record's value.
+* `value_schema_id_prefix`: Creates an Iceberg table whose structure matches the Redpanda schema for this topic, with columns corresponding to each field. You must register a schema in the Schema Registry (see next step), and producers must write to the topic using the Schema Registry wire format. Redpanda parses the schema used by the record based on the schema ID encoded in the payload header, and stores the topic values in the corresponding table columns. 
+* `disabled` (default): Disables writing to an Iceberg table for this topic.
+--
++
+[,bash]
+----
+rpk topic alter-config <new-topic-name> --set redpanda.iceberg.mode=<topic-iceberg-mode>
+----
++
+[,bash,role=no-copy]
+----
+TOPIC              STATUS
+<new-topic-name>   OK
+----
+
+. Register a schema for the topic. This step is required for the `value_schema_id_prefix` mode, but is optional otherwise.
++
+[,bash]
+----
+rpk registry schema create <subject-name> --schema </path-to-schema> --type <format>
+----
++
+[,bash,role=no-copy]
+----
+SUBJECT          VERSION   ID   TYPE
+<subject-name>   1         1    PROTOBUF
+----
+
+ifdef::env-byoc[]
+To query the Iceberg table, you need access to the object storage bucket or container where the Iceberg data is stored. For BYOC clusters on AWS and GCP, the bucket name and table location are as follows:
+
+|===
+| Bucket name | Iceberg table location
+
+| `redpanda-cloud-storage-<cluster-id>`
```

Comment:
@simon0191 could you confirm that the names and locations are the same for Azure?